src/libstd/lib.rs

```diff
@@ -551,3 +552,9 @@ include!("primitive_docs.rs");
 // the rustdoc documentation for the existing keywords. Using `include!`
 // because rustdoc only looks for these modules at the crate level.
 include!("keyword_docs.rs");
+
+// This is required to avoid an unstable error when `restricted-std` is not
```

Comment:
Is the warning about an unused feature? Could that crate instead use `#![allow(unused_features)]` instead of this workaround?

Reply:
It is a [hard error](https://github.com/rust-lang/rust/blob/0c03aee8b81185d65b5821518661c30ecdb42de5/src/librustc_passes/stability.rs#L656) when specifying an unknown feature.

Reply:
Bummer :(